src/QMCWaveFunctions/tests/test_polynomial_eeI_jastrow.cpp

```diff
@@ -179,13 +179,22 @@ TEST_CASE("PolynomialFunctor3D Jastrow", "[wavefunction]")
   std::vector<WaveFunctionComponent::RealType> dlogpsi;
   std::vector<WaveFunctionComponent::RealType> dhpsioverpsi;
 
+  std::vector<WaveFunctionComponent::ValueType> cdlogpsi;
+  std::vector<WaveFunctionComponent::ValueType> cdhpsioverpsi;
+
   psi.checkInVariables(optvars);
   optvars.resetIndex();
   const int NumOptimizables(optvars.size());
   psi.checkOutVariables(optvars);
   dlogpsi.resize(NumOptimizables);
   dhpsioverpsi.resize(NumOptimizables);
-  psi.evaluateDerivatives(elec_, optvars, dlogpsi, dhpsioverpsi);
+  cdlogpsi.resize(NumOptimizables);
```

Comment:
Please remove cdlogpsi and cdhpsioverpsi and change dlogpsi and dhpsioverpsi to ValueType and use ComplexApprox in the check.

Reply:
Fixed.